2024/calcite-react/demo/src/main.tsx

```diff
@@ -1,28 +1,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 
+import esriConfig from '@arcgis/core/config';
+
 import App from './App.tsx';
 
-// Import CSS
+// Step 1: Import CSS
 import './index.css';
 
-// JSON containing values to configure app
+// Step 3 (OPTIONAL): JSON containing values to configure app
 import {
   portalUrl,
   webmap,
   title,
   panelHeading,
 } from './config/application.json';
 
-// Calcite Components Assets (Recommended to use CDN based on documentation)
+// Step 4: Calcite Components Assets (Recommended to use CDN based on documentation)
 // Please note: You can also manage assets locally by copying assets directly from Calcite Components in your project.
 import { setAssetPath } from '@esri/calcite-components/dist/components/index';
 setAssetPath(
```

Comment:
is this the default URL or are users required to set it?
if it's default, then should be mark this step as optional?
(or even remove it from the code and just segregate it to the "additional resources" if it's only needed for more advanced setups)

Reply:
just tested and looks like it's required to set. I get an error if it isn't set 

![image](https://github.com/maxpatiiuk/esri-dev-summit-presentations/assets/29421380/ac46ecf0-01c7-44d9-bd3c-9259c1525f39)